CHANGELOG-1.2.md

```diff
@@ -19,6 +19,7 @@ and date `## vX.Y.Z - YYYY-MM-DD` and create a new placeholder section for  `unr
 ## unreleased
 
 * [CHANGE] #726 Upgrade to Cassandra 4.0-rc1
+* [CHANGE] #835 Use `kubectl cluster-info dump` to collect logs in test runs
```

Comment:
I think this kind of change shouldn't appear in the changelog. This is just for CI, end users shouldn't be interested.
wdyt?

Reply:
I agree. No need to put test-only changes in the changelog.

Reply:
Couldn't this info be useful for devs/contributors?

Reply:
It could, but devs can look at the commit history and the commit log would be there. I guess the thing here is the change is actually quite nice considering how were were grabbing logs prior. But I don't know if devs/contributors would be looking for that kind of information in just the changelog.
All that said, I really don't mind if it stays or is removed. In general though, I would agree with leaving test-only changes out of the changelog.